editor/editor_settings.cpp

```diff
@@ -1540,8 +1540,8 @@ Ref<Shortcut> ED_GET_SHORTCUT(const String &p_path) {
 }
 
 struct ShortcutMapping {
-	const char *path;
-	uint32_t keycode;
+	const char *path = nullptr;
+	uint32_t keycode = 0;
```

Comment:
This is actually unused and removed in #44128, so you might as well leave it out to avoid conflicts.